src/index.html

```diff
@@ -4,9 +4,414 @@
+          src="images/logo.png"
+          alt="moyo-header-left-logo"
+        >
+      </a>
+
+      <nav class="header__navbar">
+        <ul class="header__list">
+          <li class="header__item">
+            <a class = "header__link is-active" href="#apple">apple</a>
+          </li>
+
+          <li class="header__item">
+            <a class = "header__link" href="#samsung">samsung</a>
+          </li>
+
+          <li class="header__item">
+            <a class = "header__link" href="#smartphones">smartphones</a>
+          </li>
+
+          <li class="header__item">
+            <a
+              class = "header__link"
+              href="#laptopsandcomputers"
+              data-qa="nav-hover">
+              laptops & computers
+            </a>
+          </li>
+
+          <li class="header__item">
+            <a class = "header__link" href="#gadgets">gadgets</a>
+          </li>
+
+          <li class="header__item">
+            <a class = "header__link" href="#tablets">tablets</a>
+          </li>
+
+          <li class="header__item">
+            <a class = "header__link" href="#photo">photo</a>
+          </li>
+
+          <li class="header__item">
+            <a class = "header__link" href="#video">video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <article class="main__card card" data-qa="card">
+        <img
+          src="images/imac.jpeg"
+          alt="product-picture"
+          class="card__photo"
+        >
+
+        <h3 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+
+        <p class="card__number">
+          Product code: 195434
+        </p>
+
+        <div class="card__rating">
+          <div class="card__stars stars stars--4">
+            <div class="stars_star"></div>
+            <div class="stars_star"></div>
+            <div class="stars_star"></div>
+            <div class="stars_star"></div>
+            <div class="stars_star"></div>
+          </div>
+
+          <span class="card__reviews">
+            Reviews: 5
+          </span>
+        </div>
+
+        <div class="card__price">
+          <span class="card__price-text">
+            Price:
+          </span>
+
+          <span class="card__price-number">
+            $2,199
+          </span>
+        </div>
+
+        <button class="card__button" data-qa="card-hover">
+          buy
+        </button>
+      </article>
+
+      <article class="main__card card" data-qa="card">
+        <img
+          src="images/imac.jpeg"
+          alt="product-picture"
+          class="card__photo"
+        >
+
+        <h3 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+
+        <p class="card__number">
+          Product code: 195434
+        </p>
+
+        <div class="card__rating">
+          <div class="card__stars stars stars--4">
+            <div class="stars_star"></div>
```

Comment:
```suggestion
            <div class="stars__star"></div>
```
pls fix naming according to BEM

Reply:
check everywhere